src/Middleware/CallableHandler.php

```diff
@@ -12,6 +12,9 @@
 use Psr\Http\Server\MiddlewareInterface;
 use Psr\Http\Server\RequestHandlerInterface;
 
+/**
+ * Wraps a callable to be used to respond / as a middleware
```

Comment:
why multi line comment?

Reply:
Because its also the header / description o the class